src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -83,6 +83,13 @@ message ConfigProto {
   // A list of API interface configurations.
   repeated InterfaceConfigProto interfaces = 10;
 
+  // A list of resource name collection configs. This must be consistent with
+  // collections defined in the interfaces.
+  repeated CollectionConfigProto collections = 15;
```

Comment:
Does this PR deprecate the collection config at the interface level, or will they both be supported for the moment?